packages/@ember/-internals/glimmer/tests/integration/components/append-test.js

```diff
@@ -389,7 +394,9 @@ class AbstractAppendTest extends RenderingTestCase {
     assert.equal(renderer._roots.length, 0, 'released the root component');
   }
 
-  ['@test appending, updating and destroying multiple components'](assert) {
+  [`${testUnless(
+    DEPRECATIONS.DEPRECATE_COMPONENT_TEMPLATE_RESOLVING.isRemoved
+  )} appending, updating and destroying multiple components`](assert) {
```

Comment:
ditto

Reply:
the newly added test covers these use cases